wgengine/magicsock/magicsock_linux.go

```diff
@@ -38,8 +38,15 @@ const (
 	discoMinHeaderSize = len(disco.Magic) + 32 /* key length */ + disco.NonceLen
 )
 
-// Enable/disable using raw sockets to receive disco traffic.
-var debugDisableRawDisco = envknob.RegisterBool("TS_DEBUG_DISABLE_RAW_DISCO")
+var (
+	// Enable/disable using raw sockets to receive disco traffic.
+	debugDisableRawDisco = envknob.RegisterBool("TS_DEBUG_DISABLE_RAW_DISCO")
+	// Opt-in for using raw sockets to receive disco traffic; added for #13140.
+	//
+	// This envknob must be set and debugDisableRawDisco must be unset for
+	// raw disco to be enabled.
+	envknobEnableRawDisco = envknob.RegisterBool("TS_ENABLE_RAW_DISCO")
```

Comment:
add this to the natlab test too?

Reply:
Tweaked the natlab test to have an explicit "on" and "off" test, so that if we change the default in the future we don't need to update the test.